tests/common/include/tests/common/tools_single_type_navigator.inl

```diff
@@ -435,7 +437,7 @@ TEST(ALGEBRA_PLUGIN, single_type_navigator) {
     ASSERT_EQ(state.status(),
               toy_navigator::navigation_status::e_towards_object);
     ASSERT_EQ(state.candidates().size(), 6u);
-    ASSERT_EQ(state.next()->index, 238u);
+    ASSERT_EQ(state.next()->index, 684u);
```

Comment:
I am going to trust you here :)

Reply:
You can calculate it by yourself.
The detector takes the objects in the order of rectangle, cylinder, and disc for each volume.
vol 1: 1 + 2 
vol 2: 16 x 14 + 2 + 2 
vol 3: 2 + 2
vol 4: 32 x 14 + 2 + 2

Therefore, (1 + 2 + 16 x 14 + 2 + 2 + 32 x 14 = 683) is the index of inner cylinder of vol4 and 684 is the index of outer cylinder.